### ISSUE TITLE
Failure using method (v2_playbook_on_stats) in callback plugin

### ISSUE THREAD

**Issue by anton-kotik** [ansible/ansible#18161]
##### ISSUE TYPE
- Bug Report
##### COMPONENT NAME

callback plugin
##### ANSIBLE VERSION

```
2.1.2.0
```
##### CONFIGURATION

File /etc/ansible/hosts:

```
[localhost]
localhost ansible_connection=local
```
##### OS / ENVIRONMENT

Ubuntu 16.04
##### SUMMARY

I get the same warning "Failure using method (v2_playbook_on_stats) in callback plugin" after running any task or role on localhost, not only ping.
##### STEPS TO REPRODUCE

Create and run playbook ping.yml on localhost:

```

---
- name: Ping
  hosts: all
  tasks:
    - name: Test connection
      ping:
```
##### ACTUAL RESULTS

```
$ ansible-playbook ping.yml --verbose
Using /etc/ansible/ansible.cfg as config file

PLAY [Ping] ********************************************************************

TASK [setup] *******************************************************************
ok: [localhost]

TASK [Test connection] *********************************************************
ok: [localhost] => {"changed": false, "ping": "pong"}

PLAY RECAP *********************************************************************
 [WARNING]: Failure using method (v2_playbook_on_stats) in callback plugin 
(<ansible.plugins.callback.default.CallbackModule object
at 0x7f0c480bf510>): ''
```

**Comment by bcoca** [comment]
I cannot reproduce this with a fresh install, do you have a custom 'default' callback?

**Comment by ansibot** [comment]
@27cm ping, this issue is still waiting on your feedback. We will close the issue if you do not respond.
[click here for bot help](https://github.com/ansible/ansibullbot/blob/master/ISSUE_HELP.md)

**Comment by ansibot** [comment]
@27cm ping, this issue is still waiting on your feedback. We will close the issue if you do not respond.
[click here for bot help](https://github.com/ansible/ansibullbot/blob/master/ISSUE_HELP.md)

**Comment by ElijahLynn** [comment]
Also getting this in a playbook run. Will post more details if I go down that path.

**Comment by ansibot** [comment]
@27cm This issue is waiting for your response. Please respond or the issue will be closed.

[click here for bot help](https://github.com/ansible/ansibullbot/blob/master/ISSUE_HELP.md)

**Comment by ansibot** [comment]
@27cm This issue is waiting for your response. Please respond or the issue will be closed.

[click here for bot help](https://github.com/ansible/ansibullbot/blob/master/ISSUE_HELP.md)

**Comment by ElijahLynn** [comment]
Darn, I somehow got passed this issue but have no recollection of what was done to do that. I don't have anymore info unfortunately.

**Comment by ansibot** [comment]
@27cm You have not responded to information requests in this issue so we will assume it no longer affects you. If you are still interested in this, please create a new issue with the requested information.

[click here for bot help](https://github.com/ansible/ansibullbot/blob/master/ISSUE_HELP.md)
